server/server.go

```diff
@@ -181,19 +181,36 @@ type Server struct {
 
 // New makes a new Server
 func New(cfg Config) (*Server, error) {
+	// If user doesn't supply a logging implementation, by default instantiate
+	// logrus.
+	log := cfg.Log
+	if log == nil {
+		log = logging.NewLogrus(cfg.LogLevel)
+	}
+
```

Comment:
Why did this move?

I can guess you moved it for consistency, but you didn't mention this in the description so I have to doubt whether I understand.

Reply:
Yes, it was simply moved for consistency. Will update the PR description 👍